README.md

```diff
@@ -215,18 +278,20 @@ NOTE: Be sure to merge the latest from "upstream" before making a pull request!
+| Version | Changes                                                                                                                                                   |
+|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 1.0.0   | Initial version of SendEmail.                                                                                                                             |
+| 1.1.23  | Added FetchExchangeAttachment.                                                                                                                            |
+| 1.2.0   | Tasks no longer use Frends.Task.Attributes.                                                                                                               |
+| 1.3.0   | Fixed nuspec and references for build server.                                                                                                             |
+| 1.4.0   | ReadEmail can now read attachments. Removed FetchExchangeAttachment.                                                                                      |
+| 2.0.0   | Split ReadEmail to ReadEmailWithIMAP and ReadEmailFromExchangeServer. Added IgnoreAttachments and Office356 option.                                       |
+| 2.1.0   | Add support for Cc and Bcc recipients to SendEmail task.                                                                                                  |
+| 2.2.0   | Ported version for linux agent.                                                                                                                           |
+| 2.3.0   | Tighter MAilKit dependency rules.                                                                                                                         |
+| 2.4.0   | Now when Windows Authentication is disabled empty credentials are not set and thus frends Agent's credentials are used.                                   |
+| 2.5.0   | Changed the EmailTask to use only MailKit and replaced System.Net.Mail SmtpClient to MailKit.Net.Smtp SmtpClient.                                         |
+| 2.5.1   | Changed EmailTask to not need credentials and added selector for SecureSocketOptions and removed UseSsl bool attribute                                    |
+| 2.5.2   | Fixed bug in SendEmail task. Now email message can contain curly brackets.                                                                                |
+| 2.6.0   | Added new task for sending email to Exchange server. Added possibility to use access token as authentication method for ReadEmailFromExchangeServer-task. |
+| 3.0.0   | ReadEmailFromExchangeServer modified to use Microsoft Graph. Added task for reading sending emails to Exchange.                                           |
```

Comment:
Can you maybe expand a bit also why major version changed, so that users are aware of the actual breaking change and how to handle it?.. Just an additional sentense on that.